GiEnJul/ClientApp/src/components/admin/common/DataTableRecipient.tsx

```diff
@@ -0,0 +1,88 @@
+import * as React from "react";
```

Comment:
If DataTable is renamed, DataTableRecipient should also follow the naming pattern.